DirectEmployers/MyJobs
# Pd 3454 fix breadcrumbs with repeated values

## What's this PR do?
Fixed breadcrumb navigation by setting breadcrumbs to a unique id.

## Screenshots (if appropriate)

## Where should the reviewer start?
1. tab-actions.js
2. table-filter-actions.js
3. breadcrumb.js
4. main-reducer.js 

## How should this be manually tested?
Analytics > Click until breadcrumbs you have breadcrumbs with the same value (ie: two  "MT" breadcrumbs) 

This fix should allow the user to click between breadcrumbs, with the same name, without confusing the tabs or the breadcrumbs. 

## Any background context you want to provide?
